package.json

```diff
@@ -14,6 +15,9 @@
   },
   "devDependencies": {
     "@google/clasp": "^1.4.1",
+    "eslint": "^5.15.1",
+    "eslint-config-google": "^0.12.0",
+    "eslint-plugin-async-await": "^0.0.0",
```

Comment:
async/await isn't available in Apps Script, is it?

Reply:
Removed.